guides/session.rst

```diff
@@ -1,76 +1,114 @@
 Controlling the Browser
 =======================
 
-Ok. Now we know how to create the browser driver to talk with a specific
-browser emulator. Although we can use drivers directly to call some actions
-on the emulator, Mink provides a better way - ``Session``:
+In Mink, the entry point to the browser is called the session. Think about
+it as being your browser window (some drivers even allow to deal with switching
```

Comment:
I'm actually following the rules used in the Symfony documentation here. They are using ``you`` to address, but they are never using the first person (so no ``we`` or ``let's``). See http://symfony.com/doc/current/contributing/documentation/index.html for the Symfony documentation standards